executor/prepared.go

```diff
@@ -174,8 +176,37 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		Params:        sorter.markers,
 		SchemaVersion: e.is.SchemaMetaVersion(),
 	}
+
 	prepared.UseCache = plannercore.PreparedPlanCacheEnabled() && plannercore.Cacheable(stmt)
 
+	//Handle "ignore_plan_cache()" hint
+	//If there are multiple hints, only one will take effect
+	if prepared.UseCache {
+		switch stmt.(type) {
+		case *ast.SelectStmt:
+			for _, hints := range (stmt.(*ast.SelectStmt)).TableHints {
```

Comment:
can we do this in `plannercore.Cacheable(stmt)`?

Reply:
seems good